tst/EnergyPlus/unit/Fixtures/GroundTempsFixture.hh

```diff
@@ -0,0 +1,39 @@
+#ifndef GroundTempsFixture_hh_INCLUDED
+#define GroundTempsFixture_hh_INCLUDED
+
+// Google test headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include "EnergyPlusFixture.hh"
+#include <EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.hh>
+#include <EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh>
+#include <EnergyPlus/GroundTemperatureModeling/KusudaAchenbachGroundTemperatureModel.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteDeepGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/SiteShallowGroundTemperatures.hh>
+#include <EnergyPlus/GroundTemperatureModeling/XingGroundTemperatureModel.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+
+namespace EnergyPlus {
+
+	class GroundTempsFixture : public EnergyPlusFixture
+	{
+	protected:
+		static void SetUpTestCase() { }
+		static void TearDownTestCase() { }
+		
+		virtual void SetUp() {
+			EnergyPlusFixture::SetUp();  // Sets up the base fixture first.       
+		}
+
+		virtual void TearDown() {
+			GroundTemperatureManager::clear_state();		
```

Comment:
Also, do each of the ground temperature methods need their own unit test file? You could put them in a single GroundTemperatureModels.unit.cc or something. But that is just a minor quibble. I don't really have strong feelings about that.